uwblueprint/humane-society
# Added makefile and updated whether a column could be null

## Notion ticket link

[Add Makefile](https://www.notion.so/uwblueprintexecs/Add-Makefile-fe6e8312ecbd4d2d8bfb6c998f83a8db?pvs=4)



## Implementation description
* I created a make file with important commands that included all the required targets for linting, formatting, running migrations, generating migrations, starting/stopping Docker, cleaning the database, and seeding the database, while adding additional functionality for reading all the tables as well as being able to check the data from a specific table.
* I tested every single method to ensure its functionality, but in the process I also reverted my table into an earlier more primitive format, and I had to change the column of email to allow for null values. 
* The final thing involves the seeding target, as the seeding script has not been created yet, it has been left as a comment and it echoes a dummy message when called 


## Steps to test
1.  Verify all targets are functional (all of them should work with make installed)


## Checklist
- [✅] My PR name is descriptive and in imperative tense
- [✅] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [✅ ] I have run the appropriate linter(s)
- [✅] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
